qa/logging-config/src/test/java/org/elasticsearch/common/logging/JsonLoggerTests.java

```diff
@@ -172,7 +172,7 @@ public void testDeprecatedMessageWithoutXOpaqueId() throws IOException {
         testLogger.info( DeprecatedMessage.of("someId","deprecated message1"));
         testLogger.info( DeprecatedMessage.of("","deprecated message2"));
         testLogger.info( DeprecatedMessage.of(null,"deprecated message3"));
-        testLogger.info("deprecated message4");
+//        testLogger.info("deprecated message4");
```

Comment:
I wonder if we should allow anything like this where a deprecation logger is fetched by name 
`        final Logger testLogger = LogManager.getLogger("deprecation.test");`
and used without DeprecationLogger and DeprecationMessage class